pkg/db-repo/orm.go

```diff
@@ -19,14 +19,21 @@ type OrmSettings struct {
 	Application string              `cfg:"application" default:"{app_name}"`
 }
 
+type OrmConfig struct {
```

Comment:
Usually we use the `*Settings`  suffix for these kind of structs.

Reply:
tThanks for the review, very much appreciated. Yep I know, but `OrmSettings` already exists, this is a higher level to configure which db is used explicitly.